### ISSUE TITLE
Project abandoned?

### ISSUE THREAD

**Issue by Turmfunk** [Arthur-Milchior/anki-copy-note#24]
Hi,

I was wondering, it here will be an update yet, or if the project is abandondend. Unfortunately th addon doesn't work anymore and causes problems.

Thanks.

**Comment by Arthur-Milchior** [comment]
Not abandonned. Just, I tend to work on project if either I use them or get paid for it. I've less time nowadays for other one. Sadly for this add-on, it was not one I really needed. This should now be corrected. Can you confirm?

**Comment by Turmfunk** [comment]
Yes of course. I don't need it that often too. But from time to time it comes in quite handy. 

Am 20.02.2021 um 00:56 schrieb Arthur Milchior <notifications@github.com>:

﻿
Not abandonned. Just, I tend to work on project if either I use them or get paid for it. I've less time nowadays for other one. Sadly for this add-on, it was not one I really needed. This should now be corrected. Can you confirm?
